scripts/serie_update.py

```diff
@@ -30,6 +30,7 @@ def os_cmd(cmd, cwd=None, shell=False):
     Args:
         cmd: string command to execute.
         cwd: directory where to run command
+        shell: boolean to enable command interpretation by the shell
```

Comment:
Any impact on ./update_stm32_package.py ?
Unless this is totally transparent to users in any case, please update scripts/README.rst ?

Reply:
The function was just missing the arguments doc string, so I decided to include it in this commit.
Functionally nothing changed, can I leave it as it is?